azext_iot/iothub/_help.py

```diff
@@ -309,3 +309,38 @@ def load_iothub_help():
         type: command
         short-summary: Upload a local file as a device to a pre-configured blob storage container.
     """
+
+    helps["iot edge hierarchy"] = """
+        type: group
+        short-summary: Commands to deploy and manage nested IoT Edge device hierarchies.
+    """
+
+    helps["iot edge hierarchy create"] = """
+        type: command
+        short-summary: |
+          Deploy nested edge hierarchies to an IoT Hub. This can be done with inline
+          device arguments or by supplying a nested edge configuration file in YAML or JSON format.
+          Review examples and parameter descriptions for details on how to fully utilize the operation.
+
+        examples:
+        - name: Delete all existing devices in the Hub, and create a simple nested edge device hierarchy
+            containing 2 parent devices with 1 child device each. Also specifies module deployments
+            to configure the devices and visualizes the operations as they progress.
+          text: |
+            az iot edge hierarchy create -n {hub_name} --clean --visualize
+            --device id=parent1 deployment=/path/to/parentDeployment1.json
+            --device id=child1 parent=parent1 deployment=/path/to/child_deployment1.json
+            --device id=parent2 deployment=/path/to/parentDeployment2.json
+            --device id=child2 parent=parent2 deployment=/path/to/child_deployment2.json
+
+        - name: Create a simple nested edge device hierarchy (deleting existing devices)
+            with an existing root CA, using x509 auth, and specify a custom device bundle output path.
+          text: >
+            az iot edge hierarchy create -n {hub_name} --clean --root-cert "root_cert.pem" --root-key "root_key.pem" --device-auth x509_ca
```

Comment:
I need to add a help example for extra `--device` arguments (hostname, edge_agent, container_auth)